apps/fares/lib/fare_info.ex

```diff
@@ -5,18 +5,239 @@ defmodule Fares.FareInfo do
 
   alias Fares.Fare
 
+  @fare_data [
+    %{
+      mode: :commuter,
+      zone: "zone_1a",
```

Comment:
**non-blocking, slightly nitpick:** Since these aren't coming from the CSV I think we could name these zones in a better way. Maybe in the format the `Zones.Zone` functions expect? So this could be "1A"